src/components/navigation/navigation.html

```diff
@@ -0,0 +1,95 @@
+        </nav>
+    </header> 
+    <section class="docs-container">
+        <div class="docs-sidebar-container">
+            <div class="sidebar-content">
+                <div class="sidebar-title">Components</div>
+                <div class="sidebar-contents">
+                    <ul>
+                        <li>
+                            <a href="/src/components/avatars/avatars.html">Avatar</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/alerts/alerts.html">Alert</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/badges/badges.html">Badge</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/buttons/buttons.html">Button</a> 
+                        </li>
+                        <li>
+                            <a href="">Card</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/images/images.html">Image</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/inputs/inputs.html">Input</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/ratings/ratings.html">Rating</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/text/text.html">Text Utilities</a> 
+                        </li>
+                        <li>
+                            <a class="link-active" href="/src/components/navigation/navigation.html">Navigation</a> 
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="docs-main-container">
+            <div class="component-title">Navigation</div>
+            <div class="component-subtitle">Simple Navbar</div> 
+            <div class="component-display">
+                <nav class="nav-simple">
+                    <div class="nav-logo">Logo</div>
+                    <div class="nav-search">
+                        <button class="nav-search-icon"> <i class="fas fa-search"></i> </button>
+                        <input type="text" class="nav-search-input" placeholder="Search"/>
+                    </div>
+                    <ul class="nav-list">
+                      <li><i class="fas fa-lg fa-home"></i></li>
+                      <li><i class="fab fa-lg fa-gratipay"></i></li>
+                      <li><i class="fas fa-lg fa-shopping-cart"></i></li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="code-wrapper">
+                <iframe
+                src="https://carbon.now.sh/embed/8Pz8PQKs8F1dHRiBDYiE"
+                style="width: 708px; height: 334px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
you can use width : 100% for responsiveness

Reply:
Noted